minica_api/docker_listener.py

```diff
@@ -34,7 +34,7 @@ def listen(self):
             )
 
             for route in routes:
-                domains = findall(r"Host\(`(.+?)`\)", route)
+                domains = findall(r"(h|H)ost\(`(.+?)`\)", route)
```

Comment:
This works, but wouldn't it be better to make it fully case insensitive?
```suggestion
                from re import IGNORECASE
                domains = findall(r"Host\(`(.+?)`\)", route, flags=IGNORECASE)
```

Reply:
Yeah, that'd be a lot better

Reply:
Thanks for accepting my suggestion, but it was just a suggestion, the import statement should be moved to the top, not be there inside the function.